packages/css-jss/src/createCss.test.js

```diff
@@ -10,7 +10,7 @@ describe('css-jss', () => {
 
   beforeEach(() => {
     const jss = createJss({createGenerateId})
-    css = createCss(jss)
+    css = ((createCss(jss): any): Function)
```

Comment:
Function is an alias for any. You can avoid double casting.

Reply:
yeah, probably, was more intuitive to do that, the actual reason for any here is I have a static property on that function that is only exposed for tests and I haven't figured out how to describe that properly, since you are here, can you show me a type without casting for that css value?

Reply:
This should work.
```
type Fn = {|
  (arg: number) => void,
  staticOne: number
|}
```

Reply:
there is no such syntax with function as you showed

Reply:
Sorry, corrected

Reply:
worked